integration_test/cases/migrator.exs

```diff
@@ -81,6 +86,9 @@ defmodule Ecto.Integration.MigratorTest do
 
       assert count_entries() == 1
       assert [50] = run(TestRepo, path, :up, to: 50, log: false)
+
+      purge migrations
```

Comment:
The `[0]` is there because of https://github.com/elixir-lang/ecto/blob/master/integration_test/pg/test_helper.exs#L59

We could clean up the table here explicitly in a `setup`, and ignore the migration from `test_helper.exs`.

The `purge` calls are needed because without them the compiler complains about redefining modules.